scripts/requirements.txt

```diff
@@ -63,3 +63,6 @@ colorama
 
 # update tornado for pw_watch
 tornado
+
+# bouffalolab
+bflb-iot-tool
```

Comment:
Please make this a requirement that is environment specific. We have examples for this using esp32/mbed/nrfconnect.

If I am working on anything else except bouffalolab, I should not need to add this to python

Reply:
Moved bflb-iot-tool to requirements.bouffalolab.txt. Could you please to check whether this is OK?